### ISSUE TITLE
Permission denied when running bundle install

### ISSUE THREAD

**Issue by davefp** [Artemmkin/infrastructure-as-code-tutorial#11]
While running through the Manual Operations section I encountered the following error when trying to run `bundle install` in the raddit project:

```
An error occurred while installing tilt (2.0.8), and Bundler cannot continue.
Make sure that `gem install tilt -v '2.0.8' --source 'https://rubygems.org/'` succeeds before
bundling.
```

running the suggested command gave me this:

```
Fetching: tilt-2.0.8.gem (100%)
ERROR:  While executing gem ... (Gem::FilePermissionError)
    You don't have write permissions for the /var/lib/gems/2.3.0 directory.
```

This is after following all the previous instructions, and indeed it also occurs when running the 'Scripts' version of the instructions so it seems to be systemic
```

**Comment by davefp** [comment]
My solution so far has been to use `rbenv` to install ruby and then alter`raddit.service` to be compatible with the rbenv environment.

While that works, it adds some unfortunate extra complexity. Hopefully there's a more straightforward way around this.

**Comment by Artemmkin** [comment]
hi @davefp, thanks for reporting the issues. I'll look into this today

**Comment by Artemmkin** [comment]
@davefp to quickly fix this, run `bundle install` with `sudo`. I updated the labs cc434ea99f8f2ff6ae21a74719b93d627f484697, this should work now. Please, let me know if you encounter any other issues

**Comment by davefp** [comment]
Running `bundle install` with sudo [isn't great](https://bundler.io/man/bundle-install.1.html#SUDO-USAGE) but I see where you're coming from. Thanks for looking into this so quickly :)

**Comment by Artemmkin** [comment]
installing ruby system-wide is also not the best idea, but since this is not the focus of this tutorial, I try to keep it as simple as possible

**Comment by Artemmkin** [comment]
I first used `rvm` to install ruby for application user as you would normally do, but it seemed to add more commands and the person who is reading it has also to look it up what `rvm` does. Same thing with `rbenv`. So I went with the most straightforward way by installing it with sudo from system's repository list. I know this process looks simplistic, but I thought it would better look simplistic than to stagger a person with all the complexities of system configuration and make him quit after reading the first lab.
